Fortinet/fortigate/ingest/parser.yml

```
@@ -337,6 +337,10 @@ stages:
       - set:
           fortinet.fortigate.tunnel.ip: "{{parsed_event.message.tunnelip}}"
         filter: "{{parsed_event.message.subtype == 'vpn' and parsed_event.message.get('tunnelip') | is_ipaddress}}"
+
+      - set:
+          fortinet.fortigate.attack_direction: "{{parsed_event.message.direction}}"
+
```

I think that `network.direction` is more suitable:

```suggestion
      - translate:
        dictionary:
          "ingoing": "inbound"
          "outgoing": "outbound"
        mapping:
          parsed_event.message.direction: network.direction
        filter: "{{parsed_event.message.get('direction') != None}}"

```

Can you also add a filter to the next block below?